source/class/cv/ui/structure/tile/Controller.js

```diff
@@ -528,7 +515,7 @@ qx.Class.define('cv.ui.structure.tile.Controller', {
       if (this.__stylings && Object.prototype.hasOwnProperty.call(this.__stylings, stylingName)) {
         return this.__stylings[stylingName].mapValue(value, store);
       }
-      return value;
+      return '';
```

Comment:
Is this correct?
What about a value with invalid style? It should be displayed, just without a style.

(Note: this is just based on reading the code. The real testing and checking whether my comment is valid is TBD)

Reply:
A correct working styling adds the resulted value as CSS-class to the html element. A non working styling must not add anything to the class Attribute, especially not the unmodified value. That behavior lead to weird visualization errors.